code/modules/mining/mine_turfs.dm

```diff
@@ -414,8 +417,12 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 		visible_message("<span class='notice'>An old dusty crate was buried within!</span>")
 		DropAbandonedCrate()
 
-	var/turf/unsimulated/floor/asteroid/N = ChangeTurf(mined_type)
-	N.fullUpdateMineralOverlays()
+
+	if(mined_type == /turf/unsimulated/floor/asteroid)
```

Comment:
Should change it to if(istype(mined_type,/turf/unsimulated/floor/asteroid)) in case somebody wants mined_type to be /turf/unsimulated/floor/asteroid/air